com.unity.render-pipelines.core/Editor/Debugging/DebugWindow.cs

```diff
@@ -545,6 +529,11 @@ public class Styles
         {
             public static float s_DefaultLabelWidth = 0.5f;
 
+            public static GUIContent windowTitle { get; } = new GUIContent("Render Pipeline Debugger");
+
+            public static GUIContent resetButtonContent { get; } = new GUIContent("Reset");
```

Comment:
You should use TrTextContent to support any future localization

Reply:
Changed, I missed that one. Thanks